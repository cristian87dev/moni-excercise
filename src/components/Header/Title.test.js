import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom'; // To prevent Link dependency error.
import renderer from 'react-test-renderer';
import HeaderTitle from './Title';

it('HeaderTitle renders without crashing', () => {
  const component = renderer.create(
    <Router>
      <HeaderTitle />
    </Router>,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
