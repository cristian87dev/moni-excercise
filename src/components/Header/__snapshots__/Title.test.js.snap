// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTitle renders without crashing 1`] = `
<h1
  className="title"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="Moni Logo"
      src="moni-logo.svg"
    />
  </a>
</h1>
`;
